fix: default terminal profile removed from profiles list

Beacon6 opened the pull request:
### Problem
The dconf read command does not return the default gnome terminal profile uuid even if it was modified by the user which results in removing it from the terminal profiles list.

### Solution
I've modified the `dconf_get` function to use `gsettings get` command instead. The proposed solution handles the default profile uuid correctly. Also removed `os` from imports since it was not used anywhere in the script.

sgoudham reviewed (changes requested):
One tiny thing, could you update the the README to mention `gsettings` instead of `dconf` 

As discussed on discord, all references to `dconf` in the script should be removed.

Thanks!

Beacon6 commented:
I've replaced all uses of `dconf` with `gsettings` as discussed and also added the change to README.

I will open a separate PR for the 'uninstall' script when I finish working on it.

Edit: the `curl` installation method in the README points to an older tag version of the `install.py` script, we should probably create a new tag and update the `curl` method URL

sgoudham reviewed (approved):
> the curl installation method in the README points to an older tag version of the install.py script, we should probably create a new tag and update the curl method URL

I'm actually not sure if we want to keep versioning this theme, I suppose there's more of a case for versioning here since the user doesn't actually transfer the files themselves but uses our script for convenience. 

I might change it to just point to the main branch or I'll push out a new tag :+1:
